packages/react/src/components/sideNavigation/SideNavigation.stories.tsx

```diff
@@ -0,0 +1,272 @@
+          id={ID.SUB_LEVEL_4}
+          href={ID.SUB_LEVEL_4}
+          label={labels.subLevel4}
+          onClick={handleClick(setActive)}
+        />
+      </SideNavigation.MainLevel>
+      <SideNavigation.MainLevel
+        active={ID.MAIN_LEVEL_3 === active}
+        id={ID.MAIN_LEVEL_3}
+        href={ID.MAIN_LEVEL_3}
+        label={labels.mainLevel3}
+        onClick={handleClick(setActive)}
+      />
+      <SideNavigation.MainLevel
+        active={ID.MAIN_LEVEL_4 === active}
+        id={ID.MAIN_LEVEL_4}
+        href={ID.MAIN_LEVEL_4}
+        label={labels.mainLevel4}
+        external
+        onClick={handleClick(setActive)}
+        withDivider
+      />
+    </SideNavigation>
+  );
+};
+
+export const WithIcons = (args) => {
+  const [active, setActive] = React.useState(ID.MAIN_LEVEL_2);
+
+  return (
+    <SideNavigation {...args}>
+      <SideNavigation.MainLevel
+        active={ID.MAIN_LEVEL_1 === active}
+        id={ID.MAIN_LEVEL_1}
+        label={labels.mainLevel1}
+        onClick={handleClick(setActive)}
+        icon={<IconHome aria-hidden />}
+      >
+        <SideNavigation.SubLevel
+          active={ID.SUB_LEVEL_1 === active}
+          id={ID.SUB_LEVEL_1}
+          href={ID.SUB_LEVEL_1}
+          label={labels.subLevel1}
+          onClick={handleClick(setActive)}
+        />
+        <SideNavigation.SubLevel
+          active={ID.SUB_LEVEL_2 === active}
+          id={ID.SUB_LEVEL_2}
+          href={ID.SUB_LEVEL_2}
+          label={labels.subLevel2}
+          onClick={handleClick(setActive)}
+        />
+      </SideNavigation.MainLevel>
+      <SideNavigation.MainLevel
+        active={ID.MAIN_LEVEL_2 === active}
+        id={ID.MAIN_LEVEL_2}
+        label={labels.mainLevel2}
+        onClick={handleClick(setActive)}
+        icon={<IconHome aria-hidden />}
+      >
+        <SideNavigation.SubLevel
+          active={ID.SUB_LEVEL_3 === active}
+          id={ID.SUB_LEVEL_3}
+          href={ID.SUB_LEVEL_3}
+          label={labels.subLevel3}
+          onClick={handleClick(setActive)}
+        />
+        <SideNavigation.SubLevel
+          active={ID.SUB_LEVEL_4 === active}
+          id={ID.SUB_LEVEL_4}
+          href={ID.SUB_LEVEL_4}
+          label={labels.subLevel4}
+          onClick={handleClick(setActive)}
+        />
+      </SideNavigation.MainLevel>
+      <SideNavigation.MainLevel
+        active={ID.MAIN_LEVEL_3 === active}
+        id={ID.MAIN_LEVEL_3}
+        href={ID.MAIN_LEVEL_3}
+        label={labels.mainLevel3}
+        onClick={handleClick(setActive)}
+        icon={<IconHome aria-hidden />}
+      />
+      <SideNavigation.MainLevel
+        active={ID.MAIN_LEVEL_4 === active}
+        id={ID.MAIN_LEVEL_4}
+        href={ID.MAIN_LEVEL_4}
+        label={labels.mainLevel4}
+        external
+        onClick={handleClick(setActive)}
+        withDivider
+      />
+    </SideNavigation>
+  );
+};
+
+export const AutoCollapseOtherOpened = (args) => {
```

Comment:
Auto collapse feature has a strange functionality where all accordions collapse when a main level link is clicked. Now, this is mostly unnoticable because the page also changes. However, I assume this also happens if the main level link is external. In this case, it would be clearly visible to the user and it looks a bit weird. I think the accordions should not collapse if another main level link (internal or external) is activated.

Reply:
I guess we could remove the whole property?

Reply:
Remove the whole `AutoCollapse` property? I would be fine with it being removed for now. It is practically only useful if the side navigation has a ton of elements (and accordions) in it. Even then, maybe the navigation hierarchy design was not so successful after all.

What does @laurakarhu think about this?

Reply:
I agree, auto collapse is not mandatory to have, at least at this stage. And like Roni said, mostly helpful for massive menus.

Reply:
- [ ] remove autoCollapse-property